qutebrowser/mainwindow/statusbar/backforward.py

```diff
@@ -0,0 +1,38 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2017 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Navigation (back/forward) indicator displayed in the statusbar."""
+
+from PyQt5.QtCore import pyqtSlot
```

Comment:
Because it's already [checked by pylint](https://travis-ci.org/qutebrowser/qutebrowser/jobs/247845780#L813) and I don't want to do ignores twice everywhere for imports which aren't actually unused.